index.html

```diff
@@ -681,64 +680,58 @@ <h4><dfn>Update the EditContext</dfn></h4>
                         <dd>None</dd>
                     </dl>
             <ol>
-                <li>If the [=active EditContext=] != cached active EditContext
```

Comment:
It should be |editContext|? What is the "cached active edit context"?

Reply:
This PR removes the check against `cached active EditContext` because the |editContext| getting passed to this function is always an active EditContext.

Reply:
oh my bad :)

Reply:
Yeah I also really dislike the GitHub diff view :)